components/friend.php

```diff
@@ -303,8 +312,8 @@ public function generate( $args, $assoc_args ) {
 		for ( $i = 0; $i < $assoc_args['count']; $i++ ) {
 
 			$member = $this->get_random_user_id();
```

Comment:
Maybe use `if/else` here and below to avoid the random lookup when `--initiator` or `--friend` has been provided.

Reply:
Ops! Good catch! Fixed!